src/algebra/char_p/basic.lean

```diff
@@ -473,6 +473,10 @@ begin
   rwa ← hc
 end
 
+/-- The characteristic of a finite ring cannot be zero. -/
+lemma ring_char_ne_zero_of_fintype : ring_char R ≠ 0 :=
```

Comment:
Does this work if you put it right below `char_ne_zero_of_fintype`? I just noticed we have `char_p.ring_char_ne_one` above so I think having the `char_p` prefix is ok

Reply:
It does; I've moved it (and also moved the docstring to before `char_ne_zero_of_fintype`).